docs/ref/contrib/admin/index.txt

```diff
@@ -3165,7 +3171,7 @@ to reference your :class:`AdminSite` subclass.
 
     from django.urls import path
 
-    from myapp.admin import admin_site
+    from .admin import admin_site
```

Comment:
As Tim mentioned this in the other PR, a relative import cannot be used here as this is in a different directory

Reply:
Changed it back